aztec/src/main/kotlin/org/wordpress/aztec/EnhancedMovementMethod.kt

```diff
@@ -72,14 +77,21 @@ object EnhancedMovementMethod : ArrowKeyMovementMethod() {
                 link = text.getSpans(off, off, ClickableSpan::class.java).firstOrNull { text.getSpanStart(it) == off }
             }
 
-            if (link != null && (link is AztecMediaClickableSpan || link is UnknownClickableSpan)) {
-                if (action == MotionEvent.ACTION_UP) {
+            if (link != null) {
+                if (link is AztecMediaClickableSpan || link is UnknownClickableSpan) {
                     link.onClick(widget)
+                    return true
+                } else if (link is AztecURLSpan && clickableUrlSpan) {
+                    urlClickListener?.onClick(widget, link.url) ?: link.onClick(widget)
+                    return true
                 }
-                return true
             }
         }
 
         return super.onTouchEvent(widget, text, event)
     }
+
+    interface OnUrlClickListener {
```

Comment:
All the tap (and other) listener interfaces live inside the `AztecText`. Could we move it there, for consistency?